ResidentsSocialCarePlatformApi/V1/Gateways/SocialCareGateway.cs

```diff
@@ -144,6 +144,29 @@ public Domain.CaseNoteInformation GetCaseNoteInformationById(long caseNoteId)
             return AddRelatedInformationToVisit(visitInformation);
         }
 
+        public Domain.PersonalRelationships GetPersonalRelationships(long personId)
+        {
+            var personalRelationships = _socialCareContext.PersonalRelationshipsView
+                                            .Where(personalRelationship => personalRelationship.PersonId == personId)
+                                            .ToList();
+
+            return new Domain.PersonalRelationships()
+            {
+                Parents = FilterPersonalRelationships(personalRelationships, "parents"),
+                Siblings = FilterPersonalRelationships(personalRelationships, "siblings"),
+                Children = FilterPersonalRelationships(personalRelationships, "children"),
+                Other = FilterPersonalRelationships(personalRelationships, "other")
+            };
+        }
+
+        private List<long> FilterPersonalRelationships(List<PersonalRelationshipView> personalRelationships, string category)
+        {
+            return personalRelationships
+                        .Where(personalRelationship => personalRelationship.Category == category)
```

Comment:
The data looks normalised, but could perhaps handle this in case insensitive way for safety.

Reply:
Hmmm, the thing is `category` is determined by the view. Not sure how to test this?

Reply:
Since you control the view it's OK to leave it as it is